src/explore-education-statistics-common/src/modules/table-tool/components/__tests__/DataTableCaption.test.tsx

```diff
@@ -0,0 +1,378 @@
+          }),
+        ],
+      },
+    },
+    indicators: [
+      new Indicator({
+        label: 'Authorised absence rate',
+        value: 'authAbsRate',
+        unit: '%',
+        name: 'sess_authorised_percent',
+      }),
+    ],
+    locations: [
+      new LocationFilter({
+        value: 'E92000001',
+        label: 'England',
+        level: 'country',
+      }),
+    ],
+    timePeriodRange: [
+      new TimePeriodFilter({
+        code: 'AY',
+        year: 2015,
+        label: '2015/16',
+        order: 0,
+      }),
+    ],
+  };
+
+  test('with given title', () => {
+    const { container } = render(
+      <DataTableCaption
+        id="DataTableCaption"
+        title="Test title"
+        {...generateTableTitleBasicInput}
+      />,
+    );
+
+    expect(screen.queryByText('Test title')).not.toBeNull();
+    expect(container).not.toHaveTextContent('full table title');
+  });
+
+  test('with multiple filters labelled "Total"', () => {
+    const props = {
+      ...generateTableTitleBasicInput,
+      filters: {
+        ...generateTableTitleBasicInput.filters,
+        'School Type': {
+          name: 'school_type',
+          options: [
+            new CategoryFilter({
+              value: 'total',
+              label: 'Total',
+              category: 'School Type',
+            }),
+          ],
+        },
+      },
+    };
+    const { container } = render(
+      <DataTableCaption id="DataTableCaption" {...props} />,
+    );
+
+    expect(container).toMatchInlineSnapshot(`
+      <div>
+        <strong
+          id="DataTableCaption"
+        >
+          Table showing Authorised absence rate for 'Absence by characteristic' from 'Pupil absence in schools in England' in England for 2015/16
+        </strong>
+      </div>
+    `);
+  });
+
+  test('with multiple filters', () => {
+    const props = {
+      ...generateTableTitleBasicInput,
+      filters: {
+        ...generateTableTitleBasicInput.filters,
+        'School Type': {
+          name: 'school_type',
+          options: [
+            new CategoryFilter({
+              value: 'gender_female',
+              label: 'Female',
+              group: 'Gender',
+              category: 'Characteristic',
+            }),
+            new CategoryFilter({
+              value: 'total',
+              label: 'Total',
+              category: 'School Type',
+            }),
+            new CategoryFilter({
+              value: 'school_primary',
+              label: 'State-funded primary',
+              category: 'School Type',
+            }),
+            new CategoryFilter({
+              value: 'school_secondary',
+              label: 'State-funded secondary',
+              category: 'School Type',
+            }),
+          ],
+        },
+      },
+    };
+    const { container } = render(
+      <DataTableCaption id="DataTableCaption" {...props} />,
+    );
+
+    expect(container).toMatchInlineSnapshot(`
+      <div>
```

Comment:
Same as above, use a test id to get the element. Same applies for rest of tests.